ophyd/status.py

```diff
@@ -93,6 +93,9 @@ def _settle_then_run_callbacks(self, success=True):
                 self._cb = None
 
     def _finished(self, success=True, **kwargs):
+        if self.done:
+            return
```

Comment:
I think the `done` init kwarg was only to allow object duplication via `repr`. It doesn't appear that it's used otherwise. I think that `_finished()` definitely should only be called once.

The reset in `super.init` is a bug due to re-arranging `__init__` a bit - will fix.